src/html/popup.html

```diff
@@ -45,6 +45,14 @@
 
         <main>
             <div class="container">
+                <div id = "rateSection">
+                    <div class="section">
```

Comment:
not sure if you need two separate divs, but you could probably include the `id="rateSection"` in the second div